static/templates/left_sidebar.hbs

```diff
@@ -57,6 +57,7 @@
                     <span>{{t 'Drafts' }}</span>
                     <span class="unread_count"></span>
                 </a>
+                <span class="arrow drafts-sidebar-menu-icon"><i class="zulip-icon zulip-icon-ellipsis-v-solid" aria-hidden="true"></i></span>
```

Comment:
This is a preexisting issue, but I think these shouldn't have `aria-hidden`.  Opened https://github.com/zulip/zulip/issues/20284 to track that.

Reply:
Okay, i you opened a separate issue for that, should i just remove this label here?

Reply:
I had meant to merge this at the time of that comment, but didn't quite get a chance to finish.  I am not sure exactly how we want to fix #20284, so it seemed better to merge this as written and leave that orthogonal issue for later.